src/EditorFeatures/Core/TaskList/ITaskListProvider.cs

```diff
@@ -7,23 +7,24 @@
 using System;
 using System.Collections.Immutable;
 using System.Threading;
+using Microsoft.CodeAnalysis.TaskList;
 using Microsoft.CodeAnalysis.TodoComments;
 
-namespace Microsoft.CodeAnalysis.Editor
+namespace Microsoft.CodeAnalysis.Editor.TaskList
 {
     /// <summary>
     /// Returns Roslyn todo list from the workspace.
     /// </summary>
-    internal interface ITodoListProvider
+    internal interface ITaskListProvider
```

Comment:
One main thrust of this PR is that it unifies on the VS naming for this feature.  Namely 'Task List'.